crates/conjure_core/Cargo.toml

```diff
@@ -26,6 +26,9 @@ schemars = "0.8.21"
 clap = { version = "4.5.20", features = ["derive"] }
 itertools = "0.13.0"
 im = "15.1.0"
+sat_rs = { version = "0.1.0", path = "../../solvers/sat_rs" }
```

Comment:
I thought we decided to remove the sat_rs crate, is it still needed?

Reply:
still being used for some of the model loading step, so retaining it.